src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -101,8 +102,8 @@ public String updateUserPassword(@Valid @RequestBody UserChangePasswordDto userC
             @ApiResponse(code = 403, message = "FORBIDDEN"),
             @ApiResponse(code = 409, message = "CONFLICT")})
     @ResponseStatus(HttpStatus.ACCEPTED)
-    public String updateUserEmail(@Valid @RequestBody UserChangeEmailDto userChangeEmailDto, Principal principal)
-            throws DataConflictException {
+    public String updateUserEmail(@Valid @RequestBody UserChangeEmailDto userChangeEmailDto, BindingResult bindingResult,
```

Comment:
Could you explain why do we need BindingResult here?

Reply:
I'd like to discuss this.